core/src/test/java/org/infinispan/api/NonTxCacheTest.java

```diff
@@ -58,7 +56,7 @@ public void testRemove() throws Exception {
 
    @Override
    protected void createCacheManagers() throws Throwable {
-      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(cacheMode, false);
+      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(CacheMode.REPL_SYNC, false);
```

Comment:
Maybe you could change the test name to make it more specific to the bug it's testing?